src/PhpDoc/LattePhpDocResolver.php

```diff
@@ -0,0 +1,62 @@
+<?php
+
+namespace Efabrica\PHPStanLatte\PhpDoc;
+
+use PhpParser\Node;
+use PHPStan\Analyser\Scope;
+use PHPStan\PhpDoc\PhpDocStringResolver;
+use PHPStan\Reflection\ReflectionProvider;
+
+class LattePhpDocResolver
```

Comment:
Add final and strict please

Reply:
Isnt there CS sniff for strict types? I think I saw it somewhere. MAybe slevomat coding standard? Maybe we should add it to automate these checks.

Reply:
FOund it https://github.com/slevomat/coding-standard/blob/master/doc/type-hints.md#slevomatcodingstandardtypehintsdeclarestricttypes-

Reply:
Yes there is some, I just can’t add it to actual standard because we use it in private projects in company. Maybe I’ll add some new “stricter” standard and use it here